bench/benchmarks/remove_paint_state.js

```diff
@@ -53,7 +53,10 @@ class RemovePaintState extends Benchmark {
                     style
                 }).then(map => {
                     this.map = map;
-                });
+                })
+                    .catch(error => {
```

Comment:
Nit: the spacing is off here

Reply:
`yarn run lint` told me to write it this way 🤷‍♂

Reply:
ok 🤷‍♂